### DISCUSSION TITLE
does it take a lot of time to get a feedback for downloading llama checkpoints?

### DISCUSSION THREAD

**Discussion by meltoner** [ggml-org/llama.cpp#244] [General]
Hi i filled the form found in https://github.com/facebookresearch/llama two time since the beginning of the week 
but have not received any email. Is such a delay something to expect or perhaps its something else .. (checked spam).
Should i have gotten immediate feedback by filling the google form or requests are manually reviewed screened and could take some time to get a reply?

**Comment by m-a-sch** [comment]
you are not alone
